examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c

```diff
@@ -617,6 +598,16 @@ void HAL_GPIO_EXTI_Callback( uint16_t GPIO_Pin )
 {
     if ( GPIO_Pin == IO_NUCLEO_BUTTON_PIN )
     {
+        is_burst_mode = !( is_burst_mode );

```

Comment:
And yes, I wasn't sure if I should add it to the button or to make a new subscription topic that can force the device to enter burst mode from the Xively backend. 

Ultimately I thought that the latter would make more sense, but decided on the button for now due to time constraints.